Launcher/src/main/java/com/toolbox/LaunchToolbox.java

```diff
@@ -20,22 +20,31 @@
 import static org.lwjgl.opengl.GL11.*;
 
 public class LaunchToolbox {
-	private static final boolean useLocal = false;
+	private static final boolean useLocal = true;
```

Comment:
Well, you forgot to switch this back i guess

Reply:
woops